pyne/mesh.py

```diff
@@ -1231,23 +1231,35 @@ def tag_cell_fracs(self, cell_fracs):
         # creat tag frame with default value
         cell_largest_frac_number = [-1] * num_vol_elements
         cell_largest_frac = [0.0] * num_vol_elements
-        voxel_cell_number = np.empty(shape=(num_vol_elements,max_num_cells),
-                                     dtype=int)
-        voxel_cell_fracs = np.empty(shape=(num_vol_elements,max_num_cells),
-                                    dtype=float)
+        if max_num_cells != 1:
```

Comment:
It is usually easier to read if conditionals are positive.  So I would have swapped the two cases and written `if max_num_cells == 1`, but that is not a super blocker here.